# This msg file specifies the attributes for cylindrical objects

int32 obj_idx
geometry_msgs/Point curr_position
geometry_msgs/Point goal_position
float32 radius
float32 height
std_msgs/ColorRGBA rgbacolor